### ISSUE TITLE
[Python] Implement new Concept Exercise: comments

### ISSUE THREAD

**Issue by aldraco** [exercism/v3#1377]
This issue describes how to implement the `comments` concept exercise for the Python track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

This exercise should demonstrate and motivate the proper use of comments in Python, according to [PEP8 Standards for comments](https://www.python.org/dev/peps/pep-0008/#comments).

## Learning objectives

* Expose the student to the existence of PEP8
* Motivate the use of standardized comments
* write a proper inline comment, paragraph length block comment, and single-line block comment 

## Out of scope

- writing the classes or modules - the exercise should center around only writing comments for the code, not any code itself.
- docstrings: the student should be exposed to docstrings (perhaps in the after.md?), but this exercise does not teach or test knowledge of them.

## Concepts

 - `comments`

## Prerequisites

None. This can be done with writing no code at all.

## Resources to refer to

[PEP8 Standards for comments](https://www.python.org/dev/peps/pep-0008/#comments)

### Hints

 - Hints should link to the PEP8 documentation.

### After

After completing this exercise would be a good time to motivate docstrings, and their role in consistency/readability as well as testability in Python.

[PEP 257 for docstrings](https://www.python.org/dev/peps/pep-0257/)

## Representer

n/a

## Analyzer

n/a

## Implementing

Tests should be written using `unittest.TestCase`, and the test file named `comments_test.py`.

As far as testing something that has no runnable code, these might be some helpful places to start:

https://stackoverflow.com/questions/34511673/extracting-comments-from-python-source-code

https://pypi.org/project/comment-parser/

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

**Comment by BethanyG** [comment]
Propose that we close this in favor of folding comments into the `basics` exercise (see issue #1999 and related PR).  @cmccandless @aldraco -- and thoughts/objections here?

**Comment by cmccandless** [comment]
> Propose that we close this in favor of folding comments into the basics exercise (see issue #1999 and related PR).

Agreed.
